evm-ds/log4rs-local.yml

```diff
@@ -1,17 +1,26 @@
 appenders:
-  stderr:
+  stdout:
```

Comment:
I think stdout from evm-ds might be eaten by the node?
(although @CSideSteve recently changed the invocation mechanism and this may no longer be true).

Reply:
The output is not currently redirected, it used to be sent to /dev/null , the facility exists to redirect it to a specific place but currently stdout and stderr are not redirected for evm-ds.

Reply:
Yes, the stdout is eaten by a tail -f /dev/null in the pod configuration. we are streaming the evm-ds logs to a file and reading it back. Just set stdout for consistency with pods definition practice.

Reply:
I think stdout used to be eaten by the parent process. Maybe not anymore. But I feel like stderr is the right outptu for that.  You can e..g feed isolated server logs into one place (stdout), and evm-ds as stderr to another.

I think rather let's not change the local config?